### DISCUSSION TITLE
批量获取的url是否可以变成原始url呢

### DISCUSSION THREAD

**Discussion by lzstellata** [qiye45/wechatDownload#212] [Q&A]
现在表格中输出的url好像是转码多次后的（？），无法用于各类编辑软件的导入qwq

**Comment by qiye45** [comment]
链接是这样的
http://mp.weixin.qq.com/s?__biz=MzI4NjAxNjY4Nw==&amp;mid=2650227221&amp;idx=1&amp;sn=adc1b613061fd547f2ff67e865d0d3e2&amp;chksm=f3e0b7d5c4973ec3ace1e334232e26279f559c22862e9d5751b7f4981bfa67f43f369d24dcef&amp;scene=27#wechat_redirect

[2024-10-14-202107.csv](https://github.com/user-attachments/files/17363882/2024-10-14-202107.csv)

**Reply by qiye45** [reply]
要转一下URL编码

**Reply by lzstellata** [reply]
啊这样，那想问一下这个编码怎么转换呀

**Reply by qiye45** [reply]
简单一点，把 `&amp;` 替换成&

**Reply by lzstellata** [reply]
好的，感谢~

**Comment by lzstellata** [comment]
Thank you for your email. If your message requires a reply, I will respond as soon as possible.Best Regards LU Zhen（Stellata）
